### PR TITLE
Make docs for the cd command ; partially solves #711

### PR DESCRIPTION
**Pull request by coolshaurya**
Could have mistakes, especially regarding the default folders on Windows.

### CODE REVIEW

**Comment by coolshaurya** [comment]
Hmm... after seeing the output of `help cd` and the format on PR #746 I realize that I have to change this.

File: docs/commands/cd.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+```shell
+/home/username/Desktop/nested/folders> cd ..
+/home/username/Desktop/nested> cd ..
+/home/username/Desktop> cd ../Documents/school_related
+/home/username/Documents/school_related> cd ../../..
+/home/>
+```
+
+```shell
```
Comment (jonathandturner) [inline docs/commands/cd.md]: Looks good!

For examples 3 and 4 you could give an example of using `cd ~` to go home and `cd -` to go to the previous directory.

Comment (coolshaurya) [reply docs/commands/cd.md]: Hmm... neither of those commands work on my Manjaro Linux laptop
Screenshot from 2019-10-02 06-21-00

`nu --version` gives : 

```
nushell 0.1.3
```

Comment (andrasio) [reply docs/commands/cd.md]: @coolshaurya hi! How did you install `nu`? 

Can you run `nu` and use the `version` command? what does it say?

Comment (coolshaurya) [reply docs/commands/cd.md]: i installed nu using the `nushell-bin` AUR package

version in the nu shell gives -

```
---------
 version 
---------
 0.2.0 
---------
```

Comment (andrasio) [reply docs/commands/cd.md]: @coolshaurya Yeah, `cd ~` and `cd -` were landed after that release. This is why you are seeing errors instead.

**Pull request merged**